### ISSUE TITLE
Build Error maybe cannot find repo

### ISSUE THREAD

**Issue by heechan12** [yuyakaido/CardStackView#360]
When I tried to build, I got a error below.
"Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.yuyakaido.android:card-stack-view:2.3.4.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/yuyakaido/android/card-stack-view/2.3.4/card-stack-view-2.3.4.pom
       - https://www.jitpack.io/com/yuyakaido/android/card-stack-view/2.3.4/card-stack-view-2.3.4.pom
       - https://repo.maven.apache.org/maven2/com/yuyakaido/android/card-stack-view/2.3.4/card-stack-view-2.3.4.pom
     Required by:
         project :app
"

If I tried to check url (like https://www.jitpack.io ~~~) ,
I got "Repository not found: https://yuyakaido.com/android/card-stack-view"

<My Code>
Android Studio Version : Bumble bee
build.gradle (Project) : 
```
allprojects {
    repositories {
        google()
        maven{url 'https://www.jitpack.io'}
        mavenCentral()
    }
}
```
settings.gradle : 
```
buildscript {
    repositories {
        google()
        maven {url 'https://www.jitpack.io'}
        mavenCentral()
    }

    dependencies {
        classpath 'com.google.gms:google-services:4.3.10'
    }
}
```

**Comment by heechan12** [comment]
I solved this problem that I added "jcenter()" in gradle.

**Comment by blaticslm** [comment]
> I solved this problem that I added "jcenter()" in gradle.

Really appreciate your solution...

**Comment by saurav12bisht** [comment]
> I solved this problem that I added "jcenter()" in gradle.

yes fixed by this but jcenter() is depricated for sdk 32 do you have any other alternate .

**Comment by blaticslm** [comment]
> 

I don't have any alternative. If this exists in maven central, we wouldn't need to add jcenter to access this.

**Comment by jainishrpandya** [comment]
Found the solution. add this line in gradle.properties. -> android.enableJetifier=true
